homeassistant/components/aladdin_connect/sensor.py

```diff
@@ -42,6 +42,7 @@ class AccSensorEntityDescription(
         key="battery_level",
         name="Battery level",
         device_class=SensorDeviceClass.BATTERY,
+        entity_registry_enabled_default=False,
```

Comment:
Is there no way to detect if the device has a battery before creating the sensor entities?

Reply:
It doesn't appear to.  the JSON always has "battery_level".  I suppose we could assume 0 is no battery when we set up the integration and disable it, but 0 could also be a dead battery.
I'm open to ideas for sure.

Reply:
Ok. Let's go with the current solution for now.